src/extensions/mvs/helpers/param-definition.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2023 mol* contributors, licensed under MIT, See LICENSE file for more info.
+ *
+ * @author Adam Midlik <midlik@gmail.com>
+ */
+
+import { ParamDefinition as PD } from '../../../mol-util/param-definition';
+
+
+/** Similar to `PD.Numeric` but allows leaving empty field in UI (treated as `undefined`) */
+export function PD_MaybeInteger(defaultValue?: number, info?: PD.Info): PD.Base<number | undefined> {
+    return PD.Converted<number | undefined, PD.Text>(stringifyMaybeInt, parseMaybeInt, PD.Text(stringifyMaybeInt(defaultValue), info));
+}
+/** The magic with negative zero looks crazy, but it's needed if we want to be able to write negative numbers, LOL. Please help if you know a better solution. */
+function parseMaybeInt(input: string): number | undefined {
+    if (input.trim() === '-') return -0;
+    const num = parseInt(input);
+    return isNaN(num) ? undefined : num;
+}
+function stringifyMaybeInt(num: number | undefined): string {
+    if (num === undefined) return '';
+    if (Object.is(num, -0)) return '-';
+    return num.toString();
+}
```

Comment:
Interesting hack. I think a better solution solution could be to add `ParamDefinition.OptionalNumber`  and bake this logic into the React control itself. Though for now, this is fine too.